Makefile

```
@@ -9,41 +9,33 @@ vap_test_files:=\
-	curl -L https://github.com/kubescape/cel-admission-library/releases/latest/download/basic-control-configuration.yaml -o dist/basic-control-configuration.yaml; 
-	curl -L https://github.com/kubescape/cel-admission-library/releases/latest/download/kubescape-validating-admission-policies.yaml -o dist/validating-admission-policies.yaml; 
-
-	# Download rego rules files to dist
-	curl -L https://github.com/kubescape/regolibrary/releases/download/v2/rules -o dist/rego-rules.json; 
-
-	# Download test files from KubeScape to dist 
-	rm -f dist/vap-test-files*;
+wasm-download:
+	curl https://raw.githubusercontent.com/golang/go/refs/heads/master/lib/wasm/wasm_exec.js -o src/wasm/wasm_exec.js
+
+kubescape-cel-admission-library-download:
```

checkmake (0.3.2)

[warning] Target body for "kubescape-cel-admission-library-download" exceeds allowed length of 5 lines (9).

(maxbodylength)